src/Maker/MakeListener.php

```diff
@@ -102,6 +103,33 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
             $event = $io->askQuestion($question);
             $input->setArgument('event', $event);
         }
+
+        $event = $input->getArgument('event');
+        if (null === $this->getEventConstant($event) && null === $this->eventRegistry->getEventClassName($event)) {
+            $eventList = $this->eventRegistry->getAllActiveEvents();
+            $eventFQCNList = array_filter(array_map($this->eventRegistry->getEventClassName(...), $eventList), fn ($eventFQCN) => \is_string($eventFQCN));
+            $eventIdAndFQCNList = array_unique(array_merge($eventList, $eventFQCNList));
+            $suggestionList = [];
+            foreach ($eventIdAndFQCNList as $eventSuggestion) {
+                if (levenshtein($event, Str::getShortClassName($eventSuggestion)) < 3) {
+                    $suggestionList[] = $eventSuggestion;
+                }
+            }
+            if (!$suggestionList) {
+                return;
+            }
+            if (1 === \count($suggestionList)) {
+                $question = new ConfirmationQuestion(sprintf('<fg=green>Did you mean</> <fg=yellow>"%s"</> <fg=green>?</>', $suggestionList[0]), false);
+                $event = $io->askQuestion($question) ? $suggestionList[0] : $event;
+            } else {
```

Comment:
We can eliminate this `else` condition by returning early inside the `if` conditional. Then just have the statements within the `else` called after the `if`.